deploy/crds/kabanero_v1alpha1_kabanero_crd.yaml

```diff
@@ -87,6 +87,9 @@ spec:
                   type: boolean
               type: object
             landing:
```

Comment:
pkg/apis/kabanero/v1beta1/kabanero_types.go   already had 	Enable bool `json:"enable,omitempty"` I changed it to 	Enable *bool `json:"enable,omitempty"`     The v1alpha1/kabanero_types.go   did not have the enable bool so that is why the v1alpha1 crd was updated.      In the master branch for deploy/crds/kabanero_v1beta1_kabanero_crd.yaml I see the  landing section. Based on that and it was not updated by the tooling, I do not think I need to check in an updated kabanero_v1beta1_kabanero_crd.yaml.

Reply:
Ok, just make sure you don't have to update kabanero_kabanero_crd.yaml also.

Reply:
I updated deploy/crds/kabanero_kabanero_crd.yaml  it is now in the pull request